.bumpversion.cfg

```diff
@@ -1,5 +1,16 @@
 [bumpversion]
 current_version = 0.16.0
+commit = True
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(?:-(?P<prerel>rc)\.(?P<prerelversion>\d+))?
```

Comment:
is this also from the semvar site? could you add a comment about what its doing?

Reply:
Nope, this is my regex. I added an explanation.